Dockerfile

```diff
@@ -3,12 +3,11 @@ FROM alpine:latest as builder
 ARG KAFKA_VERSION=3.1.0
 ARG SCALA_VERSION=2.13
 
-RUN apk update && \
```

Comment:
You have removed the apk update from the Dockerfile. 

Need to add `RUN apk update && apk add --no-cache wget tar gzip` to the Dockerfile.

Reply:
noted. thanks